### PR TITLE
Update switchbot.markdown

### PR DESCRIPTION
**Pull request by alanviles**
Updated information to make it easer for people to get the BTLE MAC address and clarified some of the information that people were confused about in this https://github.com/home-assistant/home-assistant.io/issues/19135 issue

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switchbot.markdown [hunk 1]
```diff
@@ -19,6 +19,25 @@ ha_config_flow: true
...
+
+### Get the BTLE MAC address
+
+1. Open the Switchbot app.
+2. open device settings
+3. click on 'Device Info'
+
+### Note
+
+- The name in the switchbot app is not transferred to HA
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]: How about:

```suggestion
## Prerequisites

In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant. A [SwitchBot Hub](https://www.switch-bot.com/search?type=product&q=hub) is not required for this integration.

In order to set up this integration, you need to get the BTLE MAC address of your device. You can find the address for your device using the following steps:

1. Open the SwitchBot app.
2. Open device settings.
3. Click on "Device Info".
4. Write down the BLTE MAC address of your device.

Please note, device names configured in the Switchbot app are not transferred into Home Assistant.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/switchbot.markdown (hunk 1 above)
Comment (alanviles) [reply source/_integrations/switchbot.markdown]: Sounds good

File: source/_integrations/switchbot.markdown [hunk 2]
```diff
@@ -19,6 +19,18 @@ ha_config_flow: true
 ---
 
 The SwitchBot integration allows you to control SwitchBot [devices](https://www.switch-bot.com/).
+## Prerequisites
```
Comment (frenck) [inline source/_integrations/switchbot.markdown:23]:
```suggestion

## Prerequisites
```

File: source/_integrations/switchbot.markdown [hunk 3]
```diff
@@ -19,6 +19,18 @@ ha_config_flow: true
...
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant. A [SwitchBot Hub](https://www.switch-bot.com/search?type=product&q=hub) is not required for this integration.
+
+In order to set up this integration, you need to get the BTLE MAC address of your device. You can find the address for your device using the following steps:
+
+1. Open the SwitchBot app.
+2. Open device settings.
+3. Click on "Device Info".
+4. Write down the BLTE MAC address of your device.
+
+Please note, device names configured in the Switchbot app are not transferred into Home Assistant.
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]:
```suggestion
Please note, device names configured in the SwitchBot app are not transferred into Home Assistant.
```

**Review by frenck — APPROVED** [review]
Thanks, @A-Vibes 👍 

Happy Hacktoberfest 🎉 


../Frenck

**Pull request merged**